client/src/App.js

```diff
@@ -1,15 +1,12 @@
 import "./App.css";
-import Page from "../src/components/Page/Page";
+
+import SplashPage from "../src/components/Page/SplashPage";
```

Comment:
i think this shorter import should also work:

```suggestion
import SplashPage from "./components/Page/SplashPage";
```